templates/ShareCarePreview.ss

```diff
@@ -89,6 +89,7 @@
 			<% if $IncludeTwitter %><a href="https://cards-dev.twitter.com/validator" target="_blank">(debug)</a><% end_if %>
 			<span class="separator">|</span> <a href="$GooglePlusShareLink" target="_blank">Google+</a>
 			<% if $IncludePinterest %><span class="separator">|</span> <a href="$PinterestShareLink" target="_blank">Pinterest</a><% end_if %>
```

Comment:
If you don't mind could you please do me a favour and also remove the `<% if %>` wrapper around the Pinterest link here while you're at it. The Pinterest and LinkedIn share URLs should always work because they fall back to using the OG image, so there's no reason not to always display these in the CMS... this was a mistake of mine from the beginning.